<ui version="4.0" >
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>851</width>
    <height>518</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox_3" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>400</y>
     <width>331</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title" >
    <string>Quiz metadata</string>
   </property>
   <layout class="QGridLayout" >
    <item row="2" column="0" >
     <widget class="QLabel" name="label_5" >
      <property name="text" >
       <string>Number of topics</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1" >
     <widget class="LineEditWithStatus" native="1" name="m_txtNumberOfTopics" >
      <property name="enabled" >
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="1" >
     <widget class="LineEditWithStatus" native="1" name="m_txtName" />
    </item>
    <item row="1" column="0" >
     <widget class="QLabel" name="label_4" >
      <property name="text" >
       <string>Application Name</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" >
     <widget class="LineEditWithStatus" native="1" name="m_txtAuthor" />
    </item>
    <item row="0" column="0" >
     <widget class="QLabel" name="label_3" >
      <property name="text" >
       <string>Author</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_2" >
   <property name="geometry" >
    <rect>
     <x>350</x>
     <y>10</y>
     <width>491</width>
     <height>491</height>
    </rect>
   </property>
   <property name="title" >
    <string/>
   </property>
   <widget class="QListView" name="m_listTopicsSet1" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>40</y>
      <width>231</width>
      <height>411</height>
     </rect>
    </property>
   </widget>
   <widget class="QListView" name="m_listTopicsSet2" >
    <property name="geometry" >
     <rect>
      <x>260</x>
      <y>40</y>
      <width>221</width>
      <height>411</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_7" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>20</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text" >
     <string>Set 1 Topic</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2" >
    <property name="geometry" >
     <rect>
      <x>260</x>
      <y>20</y>
      <width>61</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text" >
     <string>Set2 Topic 
</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget" >
    <property name="geometry" >
     <rect>
      <x>280</x>
      <y>460</y>
      <width>196</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QToolButton" name="m_btnTopicSet2Remove" >
       <property name="text" >
        <string>...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="m_btnTopicSet2Up" >
       <property name="text" >
        <string>...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="m_btnTopicSet2Down" >
       <property name="text" >
        <string>...</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>101</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget" >
    <property name="geometry" >
     <rect>
      <x>30</x>
      <y>460</y>
      <width>196</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QToolButton" name="m_btnTopicSet1Remove" >
       <property name="text" >
        <string>...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="m_btnTopicSet1Up" >
       <property name="text" >
        <string>...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="m_btnTopicSet1Down" >
       <property name="text" >
        <string>...</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>101</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="m_groupTopicEditor" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>10</y>
     <width>331</width>
     <height>351</height>
    </rect>
   </property>
   <property name="title" >
    <string/>
   </property>
   <widget class="QTabWidget" name="tabWidget" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>20</y>
      <width>311</width>
      <height>321</height>
     </rect>
    </property>
    <property name="currentIndex" >
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab" >
     <attribute name="title" >
      <string>Set 1</string>
     </attribute>
     <widget class="QTextEdit" name="textTopicSet1" >
      <property name="geometry" >
       <rect>
        <x>50</x>
        <y>20</y>
        <width>241</width>
        <height>211</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>30</y>
        <width>47</width>
        <height>181</height>
       </rect>
      </property>
      <property name="text" >
       <string>  Topic</string>
      </property>
     </widget>
     <widget class="QPushButton" name="m_btnAddTopicSet1" >
      <property name="geometry" >
       <rect>
        <x>120</x>
        <y>270</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text" >
       <string>Add</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget" >
      <property name="geometry" >
       <rect>
        <x>60</x>
        <y>240</y>
        <width>215</width>
        <height>22</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" >
       <item>
        <widget class="QLabel" name="label_9" >
         <property name="text" >
          <string>Correct answer</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="m_txtTopicSet1Answer" />
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2" >
     <attribute name="title" >
      <string>Set 2</string>
     </attribute>
     <widget class="QTextEdit" name="textTopicSet2" >
      <property name="geometry" >
       <rect>
        <x>50</x>
        <y>20</y>
        <width>241</width>
        <height>211</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_6" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>20</y>
        <width>47</width>
        <height>201</height>
       </rect>
      </property>
      <property name="text" >
       <string>  Topic</string>
      </property>
     </widget>
     <widget class="QPushButton" name="m_btnAddTopicSet2" >
      <property name="geometry" >
       <rect>
        <x>130</x>
        <y>250</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text" >
       <string>Add</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LineEditWithStatus</class>
   <extends>QWidget</extends>
   <header>lineeditwithstatus.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
